QueryApp/README.md

```diff
@@ -55,11 +55,24 @@ is slightly different:
 
 Other than that, follow the instructions in "Installation" above.
 
-To get the paths right permanently, you might want to consider adding
-this in your `.bash_profile`, `.bashrc` och corresponding shell init
-script:
+The local installation method works best if the `PERL5LIB` environment variable
+(among others) is set correctly in the shell profile. A comprehensive guide how
+to do it correctly can be found in the manual page (`perldoc local::lib`,
+or online at [local::lib](https://metacpan.org/pod/local::lib)).
 
-    eval "`perl -I$HOME/perl5/lib/perl5 -Mlocal::lib`"
+Essentially, it boils down to adding the following line to your shell profile
+(e.g. `.bash_profile`, `.bashrc` for bash):
+
+    eval "$(perl -I$HOME/perl5/lib/perl5 -Mlocal::lib)"
```

Comment:
Why is the bashism necessary? For a simple thing like this, back quote works uniformly in quite a number of shells...

Reply:
Never mind, `$()` turns out to have been adopted for POSIX, and I didn't know...